ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.11.yaml

```diff
@@ -38,8 +38,9 @@ tests:
   steps:
     cluster_profile: aws-qe
     env:
+      AWS_METADATA_SERVICE_AUTH: Required
```

Comment:
We already set the ENV in provision chain, no need to set it here any more, this is a advantage of why we introduced provision chain.

Reply:
I forgot to remove it.